website/docs/docs/build/metrics-overview.md

```diff
@@ -0,0 +1,152 @@
+Use [expression metrics](/docs/build/expr) when you're building a metric that involves a SQL expression of multiple measures.
+
+```yaml
+# Expression metric
+metrics:
+  name: revenue_usd
+  type: expr # Expression metrics allow you to pass in any valid SQL expression.
+  type_params:
+    expr: transaction_amount_usd - cancellations_usd + alterations_usd # Define the SQL expression 
+    measures: # Define all the measures that are to be used in this expression metric 
+      - transaction_amount_usd
+      - cancellations_usd
+      - alterations_usd
+```
+-->
+
+### Simple metrics
+
+[Measure proxies](/docs/build/simple) are metrics that point directly to a measure (you may think of it as a function that takes only one measure as the input). 
+
+**Note:** If you've already defined the measure using the `create_metric: True` parameter, you don't need to create measure proxies.  However, if you would like to include a constraint on top of the measure, you will need to create a simple type metric. 
+
+```yaml
+metrics: 
+# Define the reference name of the metric.  
+# This name must be unique amongst metrics and can include lowercase letters, numbers, and underscores. 
+# This name is used to call the metric from the dbt Semantic Layer API.
+  name: cancellations 
+  type: simple 
+  type_params:
+  # Specify the measure you are creating a proxy for. 
+    measure: cancellations_usd 
+  filter: | 
+    value > 100 AND user__acquisition
+```
+
+### Ratio metrics 
+
+[Ratio metrics](/docs/build/ratio) involve a numerator measure and a denominator measure. A  `constraint` string  can be applied, to both numerator and denominator, or applied separately to the numerator or denominator. 
+
+```yaml
+# Ratio Metric
+metrics:
+- name: cancellation_rate
+  owners:
+    - support@getdbt.com
+# Ratio metrics create a ratio out of two measures.
+# Define the measures from the semantic model as numerator or denominator
+  type: ratio  
+  type_params:
+    numerator: cancellations_usd
+    denominator: transaction_amount_usd
+  filter: | # add optional constraint string. This applies to both the numerator and denominator
+    is_internal = false
+
+- name: enterprise_cancellation_rate
+  owners:
+    - support@getdbt.com
+ # Ratio metrics create a ratio out of two measures. 
+ # Define the measures from the semantic model as numerator or denominator
+  type: ratio 
+  type_params:
+    numerator: 
+      name: cancellations_usd
+      filter: tier = 'enterprise' #constraint only applies to the numerator
+    denominator: transaction_amount_usd 
+  filter: | # add optional constraint string. This applies to both the numerator and denominator
+    is_internal = false
+  
+```
+
+
+### Further configuration 
+
+You can set more metadata for your metrics, which can be used by other tools later on. The way this metadata is used will vary based on the specific integration partner
+
+- **Description** &mdash;  Write a detailed description of the metric.
```

Comment:
@Jstein77 @rpourzand this sounds incomplete - is there anything add'l to add here?

Reply:
In the spec discussion, we also have "meta" and "tags" field which allow you to set assign tags and arbitrary metadata, but I'm not 100% sure if those made it in. @Jstein77  - do you know? @QMalcolm  would probably know



meta | dbt-core | Metadata associated with the metric
-- | -- | --
tags | dbt-core | Tags associated with the metric